src/NuGet.Clients/NuGet.SolutionRestoreManager/ProjectRestoreInfo3Adapter.cs

```diff
@@ -176,36 +171,30 @@ private static IReadOnlyDictionary<string, IReadOnlyList<IVsReferenceItem2>> ToI
 
             if (targetFrameworkInfo.ProjectReferences?.Count > 0)
             {
-                IReadOnlyList<IVsReferenceItem2> projectReferences = ToReferenceItemList(targetFrameworkInfo.ProjectReferences);
-                result[ProjectItems.ProjectReference] = projectReferences;
+                result[ProjectItems.ProjectReference] = ToReferenceItemList(targetFrameworkInfo.ProjectReferences);
             }
 
             if (targetFrameworkInfo.PackageReferences?.Count > 0)
             {
-                IReadOnlyList<IVsReferenceItem2>? packageReferences = ToReferenceItemList(targetFrameworkInfo.PackageReferences);
-                result[ProjectItems.PackageReference] = packageReferences;
+                result[ProjectItems.PackageReference] = ToReferenceItemList(targetFrameworkInfo.PackageReferences);
```

Comment:
Similar to my other comment, variables are free, and this makes it easier to inspect while debugging.  You didn't add any details to the PR description, which is a violation of our process, and in particular the description part of the PR template would have helped me understand why you want to do this.

What benefit are you intending to gain from this?